src/Sylius/Bundle/CoreBundle/Settings/TaxationSettingsSchema.php

```diff
@@ -64,6 +62,10 @@ public function buildForm(FormBuilderInterface $builder)
                 'required' => false,
                 'label' => 'sylius.form.settings.taxation.default_tax_zone',
             ])
+            ->add('default_tax_calculation_strategy', 'sylius_tax_calculation_strategy_choice', [
```

Comment:
It is fine if we put it on taxation settings for now, just a note that I was considering moving default tax zone to Channel, we could do the same with this one, this would be even more flexible - having different strategies for different channels.